doc/usage/extensions/doctest.rst

```diff
@@ -67,72 +83,97 @@ a comma-separated list of group names.
-     * ``<``, ``>``: Exclusive ordered comparison clause
-     * ``===``: Arbitrary equality clause.
+      The following operands are supported:
 
-     ``pyversion`` option is followed :pep:`PEP-440: Version Specifiers
-     <440#version-specifiers>`.
+      * ``~=``: Compatible release clause
+      * ``==``: Version matching clause
+      * ``!=``: Version exclusion clause
+      * ``<=``, ``>=``: Inclusive ordered comparison clause
+      * ``<``, ``>``: Exclusive ordered comparison clause
+      * ``===``: Arbitrary equality clause.
 
-     .. versionadded:: 1.6
+      ``pyversion`` option is followed :pep:`PEP-440: Version Specifiers
+      <440#version-specifiers>`.
 
-     .. versionchanged:: 1.7
+      .. versionadded:: 1.6
 
-        Supported PEP-440 operands and notations
+      .. versionchanged:: 1.7
 
-   * ``trim-doctest-flags`` and ``no-trim-doctest-flags``, a flag option,
-     doctest flags (comments looking like ``# doctest: FLAG, ...``) at the
-     ends of lines and ``<BLANKLINE>`` markers are removed (or not removed)
-     individually.  Default is ``trim-doctest-flags``.
+         Supported PEP-440 operands and notations
 
-   Note that like with standard doctests, you have to use ``<BLANKLINE>`` to
-   signal a blank line in the expected output.  The ``<BLANKLINE>`` is removed
-   when building presentation output (HTML, LaTeX etc.).
+   .. rst:directive:option:: trim-doctest-flags
+   .. rst:directive:option:: no-trim-doctest-flags
 
-   Also, you can give inline doctest options, like in doctest::
+      Whether to trim remove doctest flags (comments looking like
+      ``# doctest: FLAG, ...``) at the ends of lines and ``<BLANKLINE>`` markers
+      individually.  Default is ``trim-doctest-flags``.
 
-      >>> datetime.date.now()   # doctest: +SKIP
-      datetime.date(2008, 1, 1)
+      Note that like with standard doctests, you have to use ``<BLANKLINE>`` to
+      signal a blank line in the expected output.  The ``<BLANKLINE>`` is removed
+      when building presentation output (HTML, LaTeX etc.).
 
-   They will be respected when the test is run, but stripped from presentation
-   output.
+   .. rst:directive:option:: skipif: condition
+      :type: text
 
+      Skip the directive if the python expression *condition* is True.
+      See :ref:`skipif`.
 
 .. rst:directive:: .. testcode:: [group]
 
    A code block for a code-output-style test.
 
-   This directive supports three options:
+   .. rubric:: Options
+
+   .. rst:directive:option:: hide
+
+      Hide the code block in other builders.  By default it is shown as a
+      highlighted code block.
+
+   .. rst:directive:option:: trim-doctest-flags
+   .. rst:directive:option:: no-trim-doctest-flags
+
+      Whether to trim remove doctest flags (comments looking like
+      ``# doctest: FLAG, ...``) at the ends of lines and ``<BLANKLINE>`` markers
+      individually.  Default is ``trim-doctest-flags``.
```

Comment:
Should we have a `:default:` option to `.. rst:directive:option::`?

Reply:
Theoretically yes. In practice, I believe it's not worth it. Without checking, I assume only a minor part of options have a real default value, instead they are often flags, or the default is some form of "emtpy"/not given e.g. in the case of `:caption:`, or the default is a behavior that needs a sentence of description, and does not have a simple value to list in a `:default:` option.
So, I'm fine with writing out "Default is ..." for the cases that make sense.